pkgs/development/libraries/gobject-introspection/wrapper.nix

```diff
@@ -18,6 +18,7 @@ in
 
 (gobject-introspection-unwrapped.override args).overrideAttrs (previousAttrs: {
   pname = "gobject-introspection-wrapped";
+  depsTargetTargetPropagated = [ gobject-introspection-unwrapped ];
```

Comment:
I am still not convinced this is correct solution for the Meson issue. `gobject-introspection` in `buildInputs` only makes sense for apps (since they will need cairo’s or GLib’s typelibs). Libraries (for which the Meson’s GIR generation is relevant) should not need those.

Reply:
It would be nice to remove it but all build systems look for it with pkg-config and use the binary variables in the pc file